### PR TITLE
[RHACS] Added release notes for 3.65

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Includes:
- Release notes for 3.65
- Updated image tags
- new settings for admission control
- 1 nit fix

Preview: https://openshift-docs-git-rel-notes-65-gnelson.vercel.app/openshift-acs/master/release_notes/365-release-notes.html

NOTES:
- This PR is for RHACS docs at https://docs.openshift.com/acs/welcome/index.html
- It doesn't require any special labels or milestones

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@IAmATeaPot418 PTAL

File: release_notes/365-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+*Release date:* September 6, 2021
+
+[id="new-features"]
+== New features
+
+[id="mitre-attck-framework"]
+=== MITRE ATT&CK framework
+You can now use the link:https://attack.mitre.org/matrices/enterprise/containers/[MITRE ATT&CK Framework] to categorize policies in the {product-title}. When you xref:../operating/manage-security-policies.adoc#create-custom-policies[create custom security policies], you can now add MITRE ATT&CK framework's adversary tactics and techniques related information.
+
+[id=""]
```
Comment (mikemckiernan) [inline release_notes/365-release-notes.adoc:19]: This empty ID value seems like an oversight, but I'm open to suggestion.  Also, this is the first time that I've *not* seen `_{context}` as a suffix for the ID.  I guess it's up to you, but seems outside the doc guidelines.

File: release_notes/365-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+{product-title} (RHACS) 3.65 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* September 6, 2021
+
+[id="new-features"]
+== New features
+
+[id="mitre-attck-framework"]
+=== MITRE ATT&CK framework
+You can now use the link:https://attack.mitre.org/matrices/enterprise/containers/[MITRE ATT&CK Framework] to categorize policies in the {product-title}. When you xref:../operating/manage-security-policies.adoc#create-custom-policies[create custom security policies], you can now add MITRE ATT&CK framework's adversary tactics and techniques related information.
```
Comment (mikemckiernan) [inline release_notes/365-release-notes.adoc:17]: nit: The MITRE thing seems treated like a proper noun.  My sugg is to either repeat with the upcase F in the second instance of Framework, or replace s/MITRE ATT&CK framework's/framework's/ and refer to it simply as a framework. Up to you.

File: configuration/enable-offline-mode.adoc [hunk 3]
```diff
@@ -80,7 +80,7 @@ include::modules/upload-definitions-to-central-api-token.adoc[leveloffset=+3]
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../cli/getting-started-cli.html#cli-authentication_cli-getting-started[Authenticating using the roxctl CLI]
+* xref:../cli/getting-started-cli.adoc#cli-authentication_cli-getting-started[Authenticating using the roxctl CLI]
```
Comment (mikemckiernan) [inline configuration/enable-offline-mode.adoc:83]: Good eyes.

File: modules/secured-cluster-configuration-options-operator.adoc [hunk 4]
```diff
@@ -56,6 +56,15 @@ The default value is `false`.
 | `admissionControl.resources.requests`
 | Use this parameter to override the default resource requests for the admission controller.
 
+| `admissionControl.bypass`
+| Use this parameter to bypass the admission controller.
+
+| `admissionControl.contactImageScanners`
+| Specify `true` to enable inline scanning of images not already scanned during a deployment's admission review.
```
Comment (mikemckiernan) [inline modules/secured-cluster-configuration-options-operator.adoc:63]: nit: maybe "...of images that are not already..."  Up to you, clearly not wrong.

File: release_notes/365-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+* link:https://aws.amazon.com/rosa/[Red Hat OpenShift Service on AWS (ROSA)]
+* link:https://azure.microsoft.com/en-au/services/openshift/#overview[Azure Red Hat OpenShift]
+
+[id="admission-control-configuration"]
+=== Admission control configuration
+You can now configure the dynamic admission control settings in the {product-title} Operator. It now includes the following new admission control settings:
+
+* `admissionControl.bypass`: Use this parameter to bypass the admission controller.
+* `admissionControl.contactImageScanners`: Specify `true` to enable inline scanning of images not already scanned during a deployment's admission review.
+* `admissionControl.timeoutSeconds`: Use this parameter to specify the maximum number of seconds {product-title} should wait for an admission review before marking it as fail open.
```
Comment (mikemckiernan) [inline release_notes/365-release-notes.adoc:32]: nit: These three bullets begin with verbs, but can you revise to start them with the same verb?  The first one, `bypass`, doesn't need to be set to `true`?

Friendly reminder that if you accepted the sugg for the desc to `contactImageScanners` in the other file, consider repeating here.  I guess it's a general reminder to make the descs the same in both files.

**Review by mikemckiernan — COMMENTED** [review]
Up-to-yous, sorry for delaying you.  Aside: the "Upcoming changes" section is courageous.

File: release_notes/365-release-notes.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply release_notes/365-release-notes.adoc:19]: Similar to OCP release notes https://raw.githubusercontent.com/openshift/openshift-docs/enterprise-4.8/release_notes/ocp-4-8-release-notes.adoc I haven't used the context here.

File: release_notes/365-release-notes.adoc (hunk 2 above)
Comment (gaurav-nelson) [reply release_notes/365-release-notes.adoc]: I navigated the page and changed the term to `Matrix` based on the description at https://attack.mitre.org/matrices/enterprise/containers/

File: release_notes/365-release-notes.adoc (hunk 5 above)
Comment (gaurav-nelson) [reply release_notes/365-release-notes.adoc:32]: It looked monotonous, I don't know. I can change if you want me to. Your call.

**Comment by gaurav-nelson** [comment]
Thanks @mikemckiernan for reviewing. I've fixed some nits and left some comments. Also, if this is okay to merge, please don't merge this yet, but add a comment that its good to merge. 3.65 is not yet released.

**Comment by mikemckiernan** [comment]
Updates seem fine.

/lgtm

Not merging, as requested.

**Pull request merged**